editor.planx.uk/src/@planx/components/shared/hooks.ts

```diff
@@ -14,6 +14,11 @@ export const useTeamSlug = () => {
   return route?.data?.team;
 };
 
+export const useFlowName = () => {
+  const route = useCurrentRoute();
+  return route?.data?.flowName;
+};
```

Comment:
This is a great little refactor, should be easy now to replace with a store function in future if required 👍